scripts/build/build/targets.py

```diff
@@ -499,22 +499,32 @@ def BuildNxpTarget():
     # boards
     target.AppendFixedTargets([
         TargetPart('k32w0', board=NxpBoard.K32W0),
-        TargetPart('k32w1', board=NxpBoard.K32W1)
+        TargetPart('k32w1', board=NxpBoard.K32W1),
+        TargetPart('rw61x', board=NxpBoard.RW61X)
+    ])
+
+    # OS
+    target.AppendFixedTargets([
+        TargetPart('zephyr', os_env=NxpOsUsed.ZEPHYR).OnlyIfRe('rw61x'),
+        TargetPart('freertos', os_env=NxpOsUsed.FREERTOS).ExceptIfRe('rw61x'),
     ])
 
     # apps
     target.AppendFixedTargets([
         TargetPart('lighting', app=NxpApp.LIGHTING).OnlyIfRe('(k32w0|k32w1)'),
-        TargetPart('contact-sensor', app=NxpApp.CONTACT).OnlyIfRe('(k32w0|k32w1)')
+        TargetPart('contact-sensor', app=NxpApp.CONTACT).OnlyIfRe('(k32w0|k32w1)'),
+        TargetPart('all-clusters', app=NxpApp.ALLCLUSTERS).OnlyIfRe('rw61x'),
+        TargetPart('laundry-washer', app=NxpApp.LAUNDRYWASHER).OnlyIfRe('rw61x'),
+        TargetPart('thermostat', app=NxpApp.THERMOSTAT).OnlyIfRe('rw61x')
```

Comment:
should these regex logic be more zephyr vs freertos instead of on board?

Reply:
I'm taking in consideration that rw61x freertos support can be added to this file later. rw61x freertos supports those 3 applications, so it makes sense to use the platform name in `OnlyIfRe`